framework/src/modules/reward/endpoint.ts

```diff
@@ -12,6 +12,51 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
+import { ModuleEndpointContext } from '../..';
 import { BaseEndpoint } from '../base_endpoint';
+import { DefaultReward, EndpointInitArgs } from './types';
 
-export class RewardEndpoint extends BaseEndpoint {}
+export class RewardEndpoint extends BaseEndpoint {
+	private _brackets!: ReadonlyArray<bigint>;
+	private _offset!: number;
+	private _distance!: number;
+
+	public init(args: EndpointInitArgs) {
+		this._brackets = args.config.brackets;
+		this._offset = args.config.offset;
+		this._distance = args.config.distance;
+	}
+
+	public getDefaultRewardAtHeight(ctx: ModuleEndpointContext): DefaultReward {
+		const { height } = ctx.params;
+
+		if (typeof height !== 'number') {
+			throw new Error('Parameter height must be a number');
```

Comment:
```suggestion
			throw new Error('Parameter height must be a number.');
```

Reply:
I couldn't remember our convention for error messages, I'll add the dots as suggested

Reply:
done